x-pack/plugins/ml/server/models/data_recognizer/modules/auditbeat_process_hosts_ecs/ml/hosts_rare_process_activity_ecs.json

```diff
@@ -0,0 +1,41 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "Auditbeat Hosts: Detect rare process executions on hosts (ECS)",
+  "groups": ["auditd"],
+  "analysis_config": {
+    "bucket_span": "10m",
+    "detectors": [
+      {
+        "function": "rare",
+        "by_field_name": "process.executable",
+        "partition_field_name": "host.name"
```

Comment:
As above, maybe give this a user-friendly detector description like `rare process execution` or even just `rare process`?